src/main/java/com/exasol/adapter/dialects/db2/DB2SqlGenerationVisitor.java

```diff
@@ -155,11 +145,18 @@ public String visit(final SqlFunctionScalar function) throws AdapterException {
             return getNullZero(function, "IFNULL(");
         case DIV:
             return getDiv(function);
+        case FROM_POSIX_TIME:
+            return getFromPosixTime(function);
         default:
             return super.visit(function);
         }
     }
 
+    private String getFromPosixTime(final SqlFunctionScalar function) throws AdapterException {
+        final String argument = function.getArguments().get(0).accept(this);
+        return "ADD_SECONDS('1970-01-01 00:00:00', " + argument + ")";
```

Comment:
I think this implementation produces wrong results on timezones != UTC.
> FROM_POSIX_TIME(<number>) is equivalent to the function call ADD_SECONDS('1970-01-01 00:00:00',<number>), if the session time zone is set to UTC.
(https://docs.exasol.com/7.0/sql_references/functions/alphabeticallistfunctions/from_posix_time.htm)
If timezone is !=  UTC the Exasol `FROM_POSIX_TIME()` will produce different results.

I think in that case it's better not to implement it than to implement it in a way that can produce wrong results

Reply:
You're right. I missed that it's equal only for UTC. I'll remove the capability